# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ntp/includes-fix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


Needed to compile ntp 4.2.0 ...

  - alan.rock@glaramara.freeserve.co.uk

diff -Naur ntp-4.2.0/include/ntp_md5.h ntp-4.2.0.new/include/ntp_md5.h
--- ntp-4.2.0/include/ntp_md5.h	2003-07-17 11:27:16.000000000 +0100
+++ ntp-4.2.0.new/include/ntp_md5.h	2003-11-11 13:31:52.000000000 +0000
@@ -3,6 +3,7 @@
  */
 
 #ifdef HAVE_MD5_H
+# include <sys/types.h>
 # include <md5.h>
 #else
 # include "rsa_md5.h"


